### PR TITLE
MTV-66: RHV cold migration

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MTV-66

Preview build: https://deploy-preview-151--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/

### CODE REVIEW

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -170,18 +178,24 @@ spec:
   map:
     - destination:
         storageClass: <storage_class>
+        accessMode: ReadWriteOnce
       source:
-        id: <source_datastore_mor> <1>
+        id: <source_datastore> <1>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: ReadWriteOnce
```
Comment (apinnick) [inline documentation/modules/migrating-virtual-machines-cli.adoc:186]: @istein1  Is `ReadWriteOnce` mandatory or can the user choose other access modes?

**Comment by apinnick** [comment]
@istein1 
Ready for QE review except for network ports.

File: documentation/modules/migration-environment-requirements.adoc [hunk 2]
```diff
@@ -7,25 +7,33 @@
...
 * IP addresses, VLANs, and other network configuration settings must not be changed before or after migration. The MAC addresses of the virtual machines are preserved during migration.
-* Uninterrupted and reliable network connections between the clusters and the replication repository.
-* Network ports enabled in the firewall rules.
+* The network connections between the source environment, the {virt} cluster, and the replication repository must be reliable and uninterrupted.
+* If you are mapping multiple source and target networks, each additional {virt} network must have its own link:https://docs.openshift.com/container-platform/{ocp-version}/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html#virt-creating-network-attachment-definition[network attachment definition].
+* The firewalls must enable traffic over the following ports:
 
-[cols="1,1,1,1,2",options="header"]
+[cols="1,1,1,1,2a",options="header"]
 .Network ports required for migration
```
Comment (apinnick) [inline documentation/modules/migration-environment-requirements.adoc:37]: Need info for network ports for RHV.

File: documentation/modules/adding-source-provider.adoc [hunk 3]
```diff
@@ -30,7 +39,18 @@ $ openssl s_client \
...
+endif::[]
+ifdef::rhv[]
+. Select *Red Hat Virtualization* from the *Type* list.
+. Fill in the following fields:
+
+* *Name*: Name to display in the list of providers
+* *Hostname or IP address*: {manager} host name or IP address
+* *Username*: {manager} user
+* *Password*: {manager} password
+* *CA certificate*: CA certificate of the {manager}. You can retrieve it at `https://<www.example.com>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply documentation/modules/adding-source-provider.adoc:51]: Not sure it makes any difference but I can move to prerequisites and see how it looks.

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 4]
```diff
@@ -56,49 +54,54 @@ $ openssl s_client \
...
   namespace: {namespace}
 type: Opaque
-stringData:
-  user: <vcenter_user> <1>
-  password: <vcenter_password> <2>
-  thumbprint: <vcenter_fingerprint> <3>
+data:
+  user: <user> <1>
+  password: <password> <2>
+  cacert: <{rhv_short}_ca_certificate> <3>
```
Comment (amastbau) [inline documentation/modules/migrating-virtual-machines-cli.adoc:73]: Should be {rhv-short}

**Review by amastbau — CHANGES_REQUESTED** [review]

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 5]
```diff
@@ -56,49 +54,54 @@ $ openssl s_client \
...
 metadata:
-  name: <vmware_secret>
+  name: <secret>
   namespace: {namespace}
 type: Opaque
-stringData:
-  user: <vcenter_user> <1>
-  password: <vcenter_password> <2>
-  thumbprint: <vcenter_fingerprint> <3>
+data:
```
Comment (amastbau) [inline documentation/modules/migrating-virtual-machines-cli.adoc:70]: stingData not data

**Review by amastbau — CHANGES_REQUESTED** [review]

**Comment by apinnick** [comment]
> /lgtm but looks like we need final confirmation about the network port?

@amastbau  Yes. I can ask at tomorrow's program meeting if I do not have an answer before then.

**Pull request merged**